src/Uno.Core/Extensions/IndentedStringBuilder.cs

```diff
@@ -79,11 +79,28 @@ public virtual void AppendFormat(IFormatProvider formatProvider, string pattern,
 			_stringBuilder.AppendFormat(formatProvider, pattern.Indent(CurrentLevel), replacements);
 		}
 
+		/// <summary>
+		/// Appends a newline.
+		/// </summary>
+		/// <remarks>
+		/// This method presents correct behavior, as opposed to it's `AppendLine(String)`
+		/// overload. Therefore, this method should be used whenever a newline is desired.
+		/// </remarks>
 		public virtual void AppendLine()
 		{
 			_stringBuilder.AppendLine();
 		}
 
+		/// <summary>
+		/// Appends the given string, *without* appending a newline at the end.
+		/// </summary>
+		/// <param name="text">The string to append.</param>
+		/// <remarks>
+		/// Even though this method seems like it appends a newline, it doesn't. It would be a
+		/// huge breaking change to fix this, so we're stuck with it for now. To append a
```

Comment:
"It would be a huge breaking change to fix this, so we're stuck with it for now. " - I would remove this part, otherwise, looks good to me. Thanks for this! :)